### DISCUSSION TITLE
MoviePilot配置-Nas媒体库自动化管理工具 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#141] [Announcements]
https://blog.goalonez.site/blog/MoviePilot%E9%85%8D%E7%BD%AE-Nas%E5%AA%92%E4%BD%93%E5%BA%93%E8%87%AA%E5%8A%A8%E5%8C%96%E7%AE%A1%E7%90%86%E5%B7%A5%E5%85%B7.html 

 Goalonez Blog

**Comment by asdflpf** [comment]
看各种教程都推荐用tr保种，目前我的下载器是qb，单纯用qb保种有啥劣势吗

**Comment by Goalonez** [comment]
> @asdflpf
> 看各种教程都推荐用tr保种，目前我的下载器是qb，单纯用qb保种有啥劣势吗

也没什么问题，就是种子数多的时候性能消耗大，保种少的话无所谓。

**Comment by Chanying95** [comment]
大佬，我跟你一样也是多块盘，媒体库里面设置怎么可以设置2块盘的路径呢？

**Comment by Goalonez** [comment]
> @Chanying95
> 大佬，我跟你一样也是多块盘，媒体库里面设置怎么可以设置2块盘的路径呢？

媒体库只能设置1个路径。但是可以通过“目录监控”插件或者“文件管理”整理到媒体库去。其实我的媒体库也是多份的，盘2是自动整理的，盘1是通过自他方式整理的。只是在极影视中选中了所有媒体库，从而达到整合的效果。

**Comment by Chanying95** [comment]
> @Goalonez
> > @Chanying95
> > 大佬，我跟你一样也是多块盘，媒体库里面设置怎么可以设置2块盘的路径呢？
> 
> 媒体库只能设置1个路径。但是可以通过“目录监控”插件或者“文件管理”整理到媒体库去。其实我的媒体库也是多份的，盘2是自动整理的，盘1是通过自他方式整理的。只是在极影视中选中了所有媒体库，从而达到整合的效果。
明白了设置不影响其他盘的硬链接和刮削呗；
再咨询一个我硬链接插件设置了但是自动硬链接的时候是复制的方式这种咋解决，我看github上我的路径应该是没问题的呀；
我的是/dow/tv：/dow/links/tv

**Comment by Goalonez** [comment]
> @Chanying95
> > @Goalonez
> > > @Chanying95
> > > 大佬，我跟你一样也是多块盘，媒体库里面设置怎么可以设置2块盘的路径呢？
> > 
> > 媒体库只能设置1个路径。但是可以通过“目录监控”插件或者“文件管理”整理到媒体库去。其实我的媒体库也是多份的，盘2是自动整理的，盘1是通过自他方式整理的。只是在极影视中选中了所有媒体库，从而达到整合的效果。
> 明白了设置不影响其他盘的硬链接和刮削呗；
> 再咨询一个我硬链接插件设置了但是自动硬链接的时候是复制的方式这种咋解决，我看github上我的路径应该是没问题的呀；
> 我的是/dow/tv：/dow/links/tv
> 

是不是跨盘了？不过我看这个路径好像没啥问题。这个去官方仓库搜一下看类似问题吧。

**Comment by chkyau** [comment]
请问大佬，下载目录可以放在固态盘上吗？因为放机械盘的话硬盘就没法休眠了。我这边怎么设置那个 downloads 它都是默认到机械盘了，辅种任务也是从机械盘上传的。还有点不明白就是明明 下载有实时速度，但是正在下载那里并没有显示任务（是不是要mp 添加的下载任务才会显示，也就是带 mp 标签的种子才会显示），另外资源搜索并不会搜索出结果来，是不是需要挂梯子才行。谢谢大佬，望能解答

**Comment by Goalonez** [comment]
> @chkyau
> 请问大佬，下载目录可以放在固态盘上吗？因为放机械盘的话硬盘就没法休眠了。我这边怎么设置那个 downloads 它都是默认到机械盘了，辅种任务也是从机械盘上传的。还有点不明白就是明明 下载有实时速度，但是正在下载那里并没有显示任务（是不是要mp 添加的下载任务才会显示，也就是带 mp 标签的种子才会显示），另外资源搜索并不会搜索出结果来，是不是需要挂梯子才行。谢谢大佬，望能解答

1.启动容器的时候，把downloads映射到你的固态盘路径去就可以了。
2.需要带mp标签的才会在mp里显示，你可以手动在下载器里添加标签，让mp接管任务。
3.应该不需要，尝试把tmdb的api地址切换成文中代理的地址。当然有代理的话就最好，毕竟github更新、tg通知也需要用到。

**Comment by chkyau** [comment]
> @Goalonez
> > @chkyau
> > 请问大佬，下载目录可以放在固态盘上吗？因为放机械盘的话硬盘就没法休眠了。我这边怎么设置那个 downloads 它都是默认到机械盘了，辅种任务也是从机械盘上传的。还有点不明白就是明明 下载有实时速度，但是正在下载那里并没有显示任务（是不是要mp 添加的下载任务才会显示，也就是带 mp 标签的种子才会显示），另外资源搜索并不会搜索出结果来，是不是需要挂梯子才行。谢谢大佬，望能解答
> 
> 1.启动容器的时候，把downloads映射到你的固态盘路径去就可以了。
> 2.需要带mp标签的才会在mp里显示，你可以手动在下载器里添加标签，让mp接管任务。
> 3.应该不需要，尝试把tmdb的api地址切换成文中代理的地址。当然有代理的话就最好，毕竟github更新、tg通知也需要用到。

谢谢大佬的解答，已经研究出来了。我整个 docker （包括 qb，tr，mp ）都是部署在固态，路径也对的，问题就是 mp 的存储空间那里一直显示的是机械硬盘的空间，后来把路径设置为真实路径，存储空间就变成固态硬盘的空间了。

**Comment by heroonezz** [comment]
大佬 你的教程实在是做的好 网上搜了那么多篇  你这篇我学到最多 有一个转种的问题 日志里报错 可以帮忙看看吗        2024/3/21 14:28:28  [31mERROR:[0m   transmission.py - transmission 连接出错：can't connect to transmission daemon: HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //192.168.1.51:9000/transmission/rpc (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f76619c5590>: Failed to resolve 'http' ([Errno -2] Name or service not known)"))

**Comment by Goalonez** [comment]
> @heroonezz
> 大佬 你的教程实在是做的好 网上搜了那么多篇  你这篇我学到最多 有一个转种的问题 日志里报错 可以帮忙看看吗        2024/3/21 14:28:28  [31mERROR:[0m   transmission.py - transmission 连接出错：can't connect to transmission daemon: HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //192.168.1.51:9000/transmission/rpc (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f76619c5590>: Failed to resolve 'http' ([Errno -2] Name or service not known)"))
> 

这个我也不太清楚，没遇到过。检查下配置地址看。或者换个tr版本试试。

**Comment by heroonezz** [comment]
> @Goalonez
> > @heroonezz
> > 大佬 你的教程实在是做的好 网上搜了那么多篇  你这篇我学到最多 有一个转种的问题 日志里报错 可以帮忙看看吗        2024/3/21 14:28:28  [31mERROR:[0m   transmission.py - transmission 连接出错：can't connect to transmission daemon: HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //192.168.1.51:9000/transmission/rpc (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f76619c5590>: Failed to resolve 'http' ([Errno -2] Name or service not known)"))
> > 
> 
> 这个我也不太清楚，没遇到过。检查下配置地址看。或者换个tr版本试试。

我弄好了  配置地址里面不带http就可以了

**Comment by monsoon585** [comment]
请问PROXY_HOST那里是应该填我的NAS局域网地址:7890还是127.0.0.1:7890

**Comment by Goalonez** [comment]
> @monsoon585
> 请问PROXY_HOST那里是应该填我的NAS局域网地址:7890还是127.0.0.1:7890

NAS局域网地址:7890

**Comment by poenl** [comment]
> @Goalonez
> > @Chanying95
> > > @Goalonez
> > > > @Chanying95
> > > > 大佬，我跟你一样也是多块盘，媒体库里面设置怎么可以设置2块盘的路径呢？
> > > 
> > > 媒体库只能设置1个路径。但是可以通过“目录监控”插件或者“文件管理”整理到媒体库去。其实我的媒体库也是多份的，盘2是自动整理的，盘1是通过自他方式整理的。只是在极影视中选中了所有媒体库，从而达到整合的效果。
> > 明白了设置不影响其他盘的硬链接和刮削呗；
> > 再咨询一个我硬链接插件设置了但是自动硬链接的时候是复制的方式这种咋解决，我看github上我的路径应该是没问题的呀；
> > 我的是/dow/tv：/dow/links/tv
> > 
> 
> 是不是跨盘了？不过我看这个路径好像没啥问题。这个去官方仓库搜一下看类似问题吧。

日志显示硬链接成功，文件夹里面也有，但占用空间是两份的大小，是什么原因呢

**Comment by wuyangrong** [comment]
大佬，刷流有两个问题想请教下。
1. 你的刷流配置，排除规则填了River，有什么作用？
2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）

**Comment by Goalonez** [comment]
> @wuyangrong
> 大佬，刷流有两个问题想请教下。
> 1. 你的刷流配置，排除规则填了River，有什么作用？
> 2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）

River是当时在看大江大河，防止订阅和刷流冲突了。
是的，删除的任务不会计入刷流体积。

**Comment by wuyangrong** [comment]
> @Goalonez
> > @wuyangrong
> > 大佬，刷流有两个问题想请教下。
> > 1. 你的刷流配置，排除规则填了River，有什么作用？
> > 2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）
> 
> River是当时在看大江大河，防止订阅和刷流冲突了。
> 是的，删除的任务不会计入刷流体积。

那这样刷流任务不会自动停下来了？大佬一般是怎么使用的呢？

**Comment by Goalonez** [comment]
> @wuyangrong
> > @Goalonez
> > > @wuyangrong
> > > 大佬，刷流有两个问题想请教下。
> > > 1. 你的刷流配置，排除规则填了River，有什么作用？
> > > 2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）
> > 
> > River是当时在看大江大河，防止订阅和刷流冲突了。
> > 是的，删除的任务不会计入刷流体积。
> 
> 那这样刷流任务不会自动停下来了？大佬一般是怎么使用的呢？

刷流一般也不需要停下来吧。
我目前设置的活跃时间超过N分钟就删除任务。
不过最近也好久没刷流了，上传量够用就好。

**Comment by wuyangrong** [comment]
> @Goalonez
> > @wuyangrong
> > > @Goalonez
> > > > @wuyangrong
> > > > 大佬，刷流有两个问题想请教下。
> > > > 1. 你的刷流配置，排除规则填了River，有什么作用？
> > > > 2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）
> > > 
> > > River是当时在看大江大河，防止订阅和刷流冲突了。
> > > 是的，删除的任务不会计入刷流体积。
> > 
> > 那这样刷流任务不会自动停下来了？大佬一般是怎么使用的呢？
> 
> 刷流一般也不需要停下来吧。
> 我目前设置的活跃时间超过N分钟就删除任务。
> 不过最近也好久没刷流了，上传量够用就好。

不能停下来，保种体积的配置就没有意义了。。。
任务不是在转种之后自动删除了吗？这样的话，感觉是不是转种过滤掉刷流比较合理
新人刚加了一个新站，刷下上传考核

**Comment by Goalonez** [comment]
> @wuyangrong
> > @Goalonez
> > > @wuyangrong
> > > > @Goalonez
> > > > > @wuyangrong
> > > > > 大佬，刷流有两个问题想请教下。
> > > > > 1. 你的刷流配置，排除规则填了River，有什么作用？
> > > > > 2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）
> > > > 
> > > > River是当时在看大江大河，防止订阅和刷流冲突了。
> > > > 是的，删除的任务不会计入刷流体积。
> > > 
> > > 那这样刷流任务不会自动停下来了？大佬一般是怎么使用的呢？
> > 
> > 刷流一般也不需要停下来吧。
> > 我目前设置的活跃时间超过N分钟就删除任务。
> > 不过最近也好久没刷流了，上传量够用就好。
> 
> 不能停下来，保种体积的配置就没有意义了。。。
> 任务不是在转种之后自动删除了吗？这样的话，感觉是不是转种过滤掉刷流比较合理
> 新人刚加了一个新站，刷下上传考核

极空间这种删掉后还在回收站里的话，确实还是会导致磁盘满的情况，需要手动去清理回收站。
转种只要保证文件路径，并不会自动删除。否则种子转到tr上应该会重新再下载一遍。就变成两份了。
我一般刷流不会转钟，直接过滤掉。统一在qb里处理刷流。否则插件也统计不到。

**Comment by wuyangrong** [comment]
> @Goalonez
> > @wuyangrong
> > > @Goalonez
> > > > @wuyangrong
> > > > > @Goalonez
> > > > > > @wuyangrong
> > > > > > 大佬，刷流有两个问题想请教下。
> > > > > > 1. 你的刷流配置，排除规则填了River，有什么作用？
> > > > > > 2. 刷流用的qb，下载完自动转移做种到tr。会收到“刷流任务种子删除”提示，这样是否不会被计算到保种体积里？（我保种体积配置的200，但是刷流任务一直在跑，做了不止200的种）
> > > > > 
> > > > > River是当时在看大江大河，防止订阅和刷流冲突了。
> > > > > 是的，删除的任务不会计入刷流体积。
> > > > 
> > > > 那这样刷流任务不会自动停下来了？大佬一般是怎么使用的呢？
> > > 
> > > 刷流一般也不需要停下来吧。
> > > 我目前设置的活跃时间超过N分钟就删除任务。
> > > 不过最近也好久没刷流了，上传量够用就好。
> > 
> > 不能停下来，保种体积的配置就没有意义了。。。
> > 任务不是在转种之后自动删除了吗？这样的话，感觉是不是转种过滤掉刷流比较合理
> > 新人刚加了一个新站，刷下上传考核
> 
> 极空间这种删掉后还在回收站里的话，确实还是会导致磁盘满的情况，需要手动去清理回收站。
> 转种只要保证文件路径，并不会自动删除。否则种子转到tr上应该会重新再下载一遍。就变成两份了。
> 我一般刷流不会转钟，直接过滤掉。统一在qb里处理刷流。否则插件也统计不到。

那就是自动转移做种插件里面过滤掉刷流，自动删种插件里面应该配置qb（好像也不需要了？因为刷流插件里面可以配置删除规则）

**Comment by taotao1996** [comment]
大佬，请问下刷流配置中设置了删除规则，种子删除后，是否会删除对应文件，我目测好像没有。那怎样才能自动删除对应文件，防止硬盘被塞满

**Comment by Goalonez** [comment]
> @taotao1996
> 大佬，请问下刷流配置中设置了删除规则，种子删除后，是否会删除对应文件，我目测好像没有。那怎样才能自动删除对应文件，防止硬盘被塞满

你是极空间吗。我这边会自动删，但是会到回收站里。回收站没法设置自动清，只能自己抽时间去手动清空。在回收站的时候依然会占用空间。

**Comment by cosmos0715** [comment]
> @heroonezz
> > @Goalonez
> > > @heroonezz
> > > 大佬 你的教程实在是做的好 网上搜了那么多篇  你这篇我学到最多 有一个转种的问题 日志里报错 可以帮忙看看吗        2024/3/21 14:28:28  [31mERROR:[0m   transmission.py - transmission 连接出错：can't connect to transmission daemon: HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //192.168.1.51:9000/transmission/rpc (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f76619c5590>: Failed to resolve 'http' ([Errno -2] Name or service not known)"))
> > > 
> > 
> > 这个我也不太清楚，没遇到过。检查下配置地址看。或者换个tr版本试试。
> 
> 我弄好了  配置地址里面不带http就可以了

**Comment by cosmos0715** [comment]
终于找到解决办法了，感谢！

**Comment by KyleLyric** [comment]
感谢大佬教程

**Comment by Goalonez** [comment]
> 感谢大佬教程

嗯，订阅规则只是一个参考。具体规则还是得看自己的需求去配置。

**Comment by zhangxl01** [comment]
请教大佬，mp具体配置馒头的方法。目前mp是1.9.12版，获得了ua和令牌。rss也生成了。但是还是提示无法连接。馒头的网址用哪个才行？

**Comment by Goalonez** [comment]
> @zhangxl01
> 请教大佬，mp具体配置馒头的方法。目前mp是1.9.12版，获得了ua和令牌。rss也生成了。但是还是提示无法连接。馒头的网址用哪个才行？

我用的cc,理论上应该跟网址没关系。检查下其他配置

**Comment by zhangxl01** [comment]
> @Goalonez
> > @zhangxl01
> > 请教大佬，mp具体配置馒头的方法。目前mp是1.9.12版，获得了ua和令牌。rss也生成了。但是还是提示无法连接。馒头的网址用哪个才行？
> 
> 我用的cc,理论上应该跟网址没关系。检查下其他配置

**Comment by zhangxl01** [comment]
上午又试了下还是不行，大佬有没有配置的图像可以参考。日志里面也没有明确的报错信息

**Comment by MONTYYUAN** [comment]
大佬，看到说硬连接要在一个盘上才行，但你这个跨盘了，为什么也可以呢？我也有跨盘的需求

**Comment by Goalonez** [comment]
> @MONTYYUAN
> 大佬，看到说硬连接要在一个盘上才行，但你这个跨盘了，为什么也可以呢？我也有跨盘的需求

同一个资源的源文件和媒体库是在同一个盘上。

**Comment by 335459215** [comment]
请问最新的1.9.6版本现在加环境变量，网络代理没有反应，一直无法链接网络，但是同一个机器里的其他docker应用都可以用那个代理

**Comment by Goalonez** [comment]
> @335459215
> 请问最新的1.9.6版本现在加环境变量，网络代理没有反应，一直无法链接网络，但是同一个机器里的其他docker应用都可以用那个代理

再检查下配置看，我这边能正常代理

**Comment by zhangxl01** [comment]
配置的截图发出来看下

**Comment by suerb** [comment]
请问「整理模式」要怎么弄？我设为「移动」模式后，通过 mp 订阅下载的电影，下载完后全都没了

**Comment by Goalonez** [comment]
> @suerb
> 请问「整理模式」要怎么弄？我设为「移动」模式后，通过 mp 订阅下载的电影，下载完后全都没了

到你设置的媒体库地址去看下，应该是被移过去了。一般还是推荐硬链接，兼容保种和刮削，反正不会多占空间。

**Comment by andrew-hongguo** [comment]
需要帮助， moviepolit 安装并运行，用户已经通过iyuu认证。但是仪表盘 存储空间只有0b. 
已经加了一个存储在设定里，（全部媒体类别），（/media/data）文件管理能看到这个，目录需要大写吗？
另外，所有的站点都不可以注册--那么我们用这个还有用吗？我的资源搜索总是没数据，花了好几天时间。。。
谢谢

**Comment by Goalonez** [comment]
> @andrew-hongguo
> 需要帮助， moviepolit 安装并运行，用户已经通过iyuu认证。但是仪表盘 存储空间只有0b. 
> 已经加了一个存储在设定里，（全部媒体类别），（/media/data）文件管理能看到这个，目录需要大写吗？
> 另外，所有的站点都不可以注册--那么我们用这个还有用吗？我的资源搜索总是没数据，花了好几天时间。。。
> 谢谢

空间为0应该是你启动的时候文件夹映射关系没配对，检测不到目录。
然后你工具页面里面配置的应该是容器内部的目录地址，比如你把容器里的/media映射到了机器上的/空间/媒体库，那页面里就应该配置/media。
这个工具只是用来帮助你更方便地管理站点，它本身并没有资源，资源都来自于站点，只是它基于api帮你去站点做了相应的操作。所以必须要有站点。

**Comment by andrew-hongguo** [comment]
谢谢，目录确实是填错了，现在有了。
现在我可以搜索到大部分的电影电视剧，但是只有少数下载，但是都只下载0b -- 我需要什么设置吗？还是要注册更多的网站？或者一般这个要等很长时间?
关于网站，有没有网站而不是网名？例如，憨憨，我问了豆包和kim,都找不到网址
谢谢帮助

**Comment by bohanming** [comment]
你这个不需要代理服务器

**Comment by pototazhang** [comment]
大佬，我现在有两块16T机械硬盘，放在硬盘柜里连接的电脑，还有一块1T固态硬盘，用的是Mac电脑，我想把这个Mac当作一个NAS使用，现在遇到了一个难题，我用的是mp+qb，来实现的自动下载，那我的固态如何也能利用起来，比如qb的下载目录设置到哪里，如果设置到16T的里面，固态是不是就没法利用了，但是下载目录设置到固态里，该怎么把资源转移到16t里，硬链接不可行，因为跨磁盘了，软链接又会影响固态做种 有什么好的解决方案吗

**Comment by Goalonez** [comment]
> @pototazhang
> 大佬，我现在有两块16T机械硬盘，放在硬盘柜里连接的电脑，还有一块1T固态硬盘，用的是Mac电脑，我想把这个Mac当作一个NAS使用，现在遇到了一个难题，我用的是mp+qb，来实现的自动下载，那我的固态如何也能利用起来，比如qb的下载目录设置到哪里，如果设置到16T的里面，固态是不是就没法利用了，但是下载目录设置到固态里，该怎么把资源转移到16t里，硬链接不可行，因为跨磁盘了，软链接又会影响固态做种 有什么好的解决方案吗

这个问题我也不太清楚，可以去mp的tg群里问一下。

**Comment by sunskyhsh** [comment]
> @chkyau
> > @Goalonez
> > > @chkyau
> > > 请问大佬，下载目录可以放在固态盘上吗？因为放机械盘的话硬盘就没法休眠了。我这边怎么设置那个 downloads 它都是默认到机械盘了，辅种任务也是从机械盘上传的。还有点不明白就是明明 下载有实时速度，但是正在下载那里并没有显示任务（是不是要mp 添加的下载任务才会显示，也就是带 mp 标签的种子才会显示），另外资源搜索并不会搜索出结果来，是不是需要挂梯子才行。谢谢大佬，望能解答
> > 
> > 1.启动容器的时候，把downloads映射到你的固态盘路径去就可以了。
> > 2.需要带mp标签的才会在mp里显示，你可以手动在下载器里添加标签，让mp接管任务。
> > 3.应该不需要，尝试把tmdb的api地址切换成文中代理的地址。当然有代理的话就最好，毕竟github更新、tg通知也需要用到。
> 
> 谢谢大佬的解答，已经研究出来了。我整个 docker （包括 qb，tr，mp ）都是部署在固态，路径也对的，问题就是 mp 的存储空间那里一直显示的是机械硬盘的空间，后来把路径设置为真实路径，存储空间就变成固态硬盘的空间了。

请问大佬，怎么把路径设置为真实路径？我也遇到了mp的存储空间一直显示的是机械硬盘的空间这个问题。

**Comment by goodnightzsj** [comment]
您好，我想问下媒体整理的时候对资源的分类策略是怎么匹配自定义的二级分类策略的？我目前遇到的问题是：开心超人联盟之机械堡奇遇记这个资源期望应该被分配在动漫目录下，但目前被分类成国产电视剧了，我看docker日志好像是根据tmdb查到的结果来匹配的二级分类策略。请问我要如何做才能让资源被整理到正确的目录下呢

**Comment by Goalonez** [comment]
> @goodnightzsj
> 您好，我想问下媒体整理的时候对资源的分类策略是怎么匹配自定义的二级分类策略的？我目前遇到的问题是：开心超人联盟之机械堡奇遇记这个资源期望应该被分配在动漫目录下，但目前被分类成国产电视剧了，我看docker日志好像是根据tmdb查到的结果来匹配的二级分类策略。请问我要如何做才能让资源被整理到正确的目录下呢

应该是根据tmdb来的，但是我也没遇到过这种情况，所以不太清楚怎么操作修正。可以去tg群问一下。

**Comment by yzf867601525** [comment]
请教一下，那个我下载影视的时候老是报错：下载失败：无法识别媒体信息。这是哪一步配置错误，

**Comment by Goalonez** [comment]
> @yzf867601525
> 请教一下，那个我下载影视的时候老是报错：下载失败：无法识别媒体信息。这是哪一步配置错误，

如果下载所有的资源都报错，就检查下代理。如果只是部分影视的话尝试重启或者重新构建容器，偶尔会出现这种情况。
